core/src/integrationTest/kotlin/com/malinskiy/marathon/analytics/metrics/remote/TestEvents.kt

```diff
@@ -10,6 +10,7 @@ import com.malinskiy.marathon.execution.TestStatus
 import com.malinskiy.marathon.generateTest
 import java.time.Instant
 import java.time.temporal.ChronoUnit
+import java.util.*
```

Comment:
Maybe add detekt and suppress `star imports`

Reply:
detekt is enabled, we ignore it in the CI
IDEA codestyle is better, no will check why it hasn't taken effect here

Reply:
fixed the codestyle autoformatting and the import